map_model/src/edits/mod.rs

```diff
@@ -448,11 +448,16 @@ fn recalculate_turns(id: IntersectionID, map: &mut Map, effects: &mut EditEffect
 }
 
 fn modify_lanes(map: &mut Map, r: RoadID, lanes_ltr: Vec<LaneSpec>, effects: &mut EditEffects) {
+    let new_width = lanes_ltr
+        .iter()
+        .fold(Distance::ZERO, |sum, x| sum + x.width);
+    let old_width = 2.0 * map.get_r(r).get_half_width(map);
```

Comment:
I would like to make the incredibly bold suggestion that we introduce a `map.get_r(r).width()` which doesn't not divide the width by 2. 😉

Reply:
_Gasp_
Done.